README.md

```diff
@@ -92,7 +92,14 @@ So, we allow you to collect all the docgen info into a global collection. To do
 ```json
 {
   "plugins":[
-    ["babel-plugin-react-docgen", { "DOC_GEN_COLLECTION_NAME": "MY_REACT_DOCS"}]
+    [
+      "babel-plugin-react-docgen", 
+      { 
+        "DOC_GEN_COLLECTION_NAME": "MY_REACT_DOCS",
+        "resolver": "findAllExportedComponentDefinitions", // optional (default: undefined)
```

Comment:
Let's add a bit more documentation here on what this `findAllExportedComponentDefinitions` does.

Reply:
alright, added some info about the resolvers and where to find more background information